vm/resolutions.go

```diff
@@ -129,32 +137,23 @@ func (vm *VM) Verified(ctx context.Context, b *chain.StatefulBlock) {
 	vm.checkActivity(ctx)
 
 	if b.Processed() {
-		fm := b.FeeManager()
 		vm.snowCtx.Log.Info(
 			"verified block",
-			zap.Stringer("blkID", b.ID()),
-			zap.Uint64("height", b.Hght),
-			zap.Int("txs", len(b.Txs)),
-			zap.Stringer("parent root", b.StateRoot),
+			zap.Stringer("executedBlk", b.executedBlock),
```

Comment:
Can we make the key here "blk" to be consistent w/ the rest of the logs?

Reply:
Will do